src/components/NavBar.jsx

```diff
@@ -4,10 +4,10 @@ import './NavBar.css';
 const NavBar = () => (
   <header className="header">
     <>
-      <h1>Book store</h1>
+      <h1>Book store CMS</h1>
       <ul>
-        <li><Link to="/">Home</Link></li>
-        <li><Link to="/categories">Categories</Link></li>
+        <li><Link to="/">HOME</Link></li>
+        <li><Link to="/categories">CATEGORIES</Link></li>
       </ul>
     </>
```

Comment:
- [ ] @wacoo Great work on your navbar styling, however the text color and highlight when the user clicks on the nav items differ from the design given, The navbar is surrounded by a background margin that covers the whole page and the logo on the right side is missing,  check the comparison below

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/wacoo/react-book-store/assets/61408860/8791c89e-036c-45d4-b2da-a0a690908fd8) | ![image](https://github.com/wacoo/react-book-store/assets/61408860/dbc3c276-3351-4037-bc13-f529e0d8fad7)

Reply:
- [ ] [optional] Your navbar is only appearing on the booklist page instead of all pages, I would suggest you import your navbar to the app.js component instead of the booklist component.